schematic_api/api/openapi/api.yaml

```diff
@@ -101,8 +101,57 @@ paths:
         - Manifest Operations
   /manifest/download:
     get:
-      summary: Endpoint to download an existing manifest
-      description: Endpoint to download an existing manifest
+      summary: Endpoint to download an existing manifest by using a manifest id
+      description: Endpoint to download an existing manifest by using a manifest id
+      parameters:
+        - in: query
```

Comment:
This isn't great security practice, and I would strongly advise not adding any other features that leverages this until it is fixed.

Reply:
That makes sense. I will add a TODO here.